writers/streamingbatchwriter/streamingbatchwriter.go

```diff
@@ -1,4 +1,4 @@
-package writers
+package streamingbatchwriter
```

Comment:
Just for my own reference, what's the difference between streaming batch writer and the other two batch writers?

It would also be good to add the answer to this question as a comment in the package, as well as a similar one for BatchWriter, so users can more easily figure out when to use which one.

Reply:
@hermanschaaf added a short blurb